recipes/nyuki_recipe/meta.yaml

```diff
@@ -0,0 +1,71 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "nyuki" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source: 
+  url: https://github.com/00krishna-tools/nyuki/archive/v{{ version }}.tar.gz
+  sha256: 88e943eed59d5ca95d6716b22b47c71a7b7c9323a71348a4623e329bb66d7ae7
+
+build:
+  noarch: python
+  number: 0
+  skip: true  # [py<36]
```

Comment:
You don't need the `# [py<36]` since you're controlling the Python version with `python >3.5` below.

Reply:
@dopplershift Oh excellent. Yeah, I will make that change. Thanks for taking a look. I am just trying to test the build on different platforms before really pushing it for a real review. Let me know if you find anything else or if there is a better way for me to reduce the priority on this so that no one wastes there time :). Thanks again.

Reply:
@dopplershift this should be resolved now. I removed the selector.